apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml

```diff
@@ -123,7 +123,36 @@ Jedis:
 SQLite:
   id: 31
   languages: Java,C#
-
+h2-jdbc-driver:
+  id: 32
+  languages: Java
+mysql-connector-java:
+  id: 33
+  languages: Java
+Spymemcached:
+  id: 35
+  languages: Java
+Xmemcached:
+  id: 36
+  languages: Java
+postgresql-jdbc-driver:
+  id: 37
+  languages: Java
+rocketMQ-producer-client:
```

Comment:
producer and client are same thing for kafka

Reply:
or rocketmq

Reply:
well, How can I mapping the kafka or rocketMQ in the `Component-Server-Mappings` node?

Reply:
you should name as consumer and producer. No client needed. They are all clients for kafka server.